src/telegram_common_plug.c

```diff
@@ -32,8 +172,12 @@ int telegram_valid(char *ciphertext, struct fmt_main *self)
 		goto err;
 	if (!isdec(p))
 		goto err;
-	value = atoi(p);
-	if (value != 1)
+	version = atoi(p);
+#ifdef OPENCL_FORMAT
+	if (version != 1)
```

Comment:
Maybe we should print a warning to stderr when the OpenCL format sees and rejects version 2. IIRC, we already have something like this in some other format(s).